### PR TITLE
Separate unsafeLookupEnv function

### LINKED ISSUE #1: The environment variable can be checked multiple times
This can be easily shown by replacing `lookupEnv` with `getLine`. The environment variable is checked once per trace message (basically, once per call site), whereas it can be checked once per variable name instead.

### LINKED ISSUE #2: The environment variable value does not matter
It is only checked whether the variable is set or not.

### PR DESCRIPTION
**Pull request by int-index**
fixes #1 
fixes #2 

the documentation is slightly improved too

### CODE REVIEW

File: src/Debug/Tracy.hs [hunk 1]
```diff
@@ -1,5 +1,6 @@
 module Debug.Tracy
-  ( makeTracy
+  ( unsafeLookupEnv
```
Comment (dredozubov) [inline src/Debug/Tracy.hs]: i don't see any reason for it to be exported

File: tracy.cabal [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name:                tracy
-version:             0.1.2.0
+version:             0.1.3.0
```
Comment (dredozubov) [inline tracy.cabal]: Can you drop the version bump?

**Comment by dredozubov** [comment]
I've mentioned a few details, but otherwise i like the patch.

**Comment by int-index** [comment]
Ok, I updated the patch according to the comments.

**Pull request merged**